packages/mobile_app/lib/services/ads.dart

```diff
@@ -0,0 +1,35 @@
+part of 'static_provider.dart';
+
+RewardedAd getCardAd() => RewardedAd(
```

Comment:
Im not sure we need a `RewardAd`. Just a regular `Interstitial` may be sufficient. Lets try though